controllers/cron_controller.py

```diff
@@ -444,3 +445,12 @@ def get(self, event_key):
         EventManipulator.createOrUpdate(event)
 
         self.response.out.write(event.district_points)
+
+
+class UpcomingNotificationDo(webapp.RequestHandler):
+    """
+    Sends out notifications for upcoming matches
+    """
+    def get(self):
+        live_events = EventHelper.getEventsWithinADay()
+        NotificationHelper.send_upcoming(live_events)
```

Comment:
`send_upcoming_matches` makes more sense as a method name. Call `EventHelper.getEventsWithinADay()` within that method.